charts/csi-secrets-store-provider-azure/values.yaml

```diff
@@ -38,3 +38,5 @@ secrets-store-csi-driver:
   windows:
     enabled: false
     kubeletRootDir: C:\var\lib\kubelet
+  logLevel:
+    debug: false
```

Comment:
Should we try to be consistent here: https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/charts/secrets-store-csi-driver/values.yaml#L18

Reply:
I think we should keep `debug: false` by default for driver too as the liveness probe logs are debug level and can spam the logs periodically. I'll open a PR in the driver to set `debug: false` in the driver repo.

Reply:
I'll keep it consistent for now. We need to evaluate the current logs that are in debug and see which can be bumped up to `Info` level. I think with the next release of helm charts, we should switch `debug: false` so that driver logs aren't filled with `livenessprobe` request logs.

Reply:
sgtm